addon/templates/components/popover-on-element.hbs

```diff
@@ -1,3 +1,12 @@
-{{yield (hash
-	hide=(action "hide")
-)}}
+{{#if _shouldRender}}
+	{{#lazy-popover-on-element
```

Comment:
The `lazy-popover-on-element` is the Tether component. When this is rendered it will insert the Tether component, which will also attach the tether behavior to the target.

I'm unhappy with the name `lazy-popover-on-element`. I think I'm going to go with `popover-tether-component` and `tooltip-tether-component` to indicate the roles of the various components